### ISSUE TITLE
not work in kibana 5.2.0

### ISSUE THREAD

**Issue by xycloud** [CristianWeiland/kibana-datepicker-plugin#3]
not work in kibana 5.2.0, cannot see the datepicker in embed dashbord

**Comment by CristianWeiland** [comment]
It worked for me. Could you elaborate a bit more your problem? Did you change the kibana version in the plugin's package.json? Does it throw any kind of error?

**Comment by xycloud** [comment]
yes, i changed kibana version as 5.2.0 in package.json, and I did not see any error.

**Comment by CristianWeiland** [comment]
Were you able to create the visualization correctly and add it to your dashboard?

**Comment by Samuren111** [comment]
Hi 

Plugin is perfectly working, I had just one question what does move "go" does? 

Thanks in advance
Sam
